Tools/autotest/common.py

```diff
@@ -604,7 +605,7 @@ def log_download(self, filename, timeout=360, upload_logs=False):
         """Download latest log."""
         self.wait_heartbeat()
         self.mavproxy.send("module load log\n")
-        self.mavproxy.expect("Loaded module log")
+        self.mavproxy.expect(["Loaded module log", "module log already loaded"])
```

Comment:
This should never be the case.

Why was this required?

Whoever loaded it should unload it :-)

Reply:
Probably just debug. I'll take it out.

Reply:
No, it's required. Not sure who isn't unloading the log module - but it isn't me!